chart/keep/values.yaml

```diff
@@ -52,6 +52,12 @@ backend:
           - path: /
             pathType: ImplementationSpecific
     tls: []
+  route:
+    enabled: false
+    host: chart-example-backend.local
```

Comment:
what's that exactly? why is it chart-example-backend and not keep-backend?

Reply:
taken from your ingress host value as to not deviate.